scapy/arch/windows/__init__.py

```diff
@@ -393,8 +393,12 @@ def _check_npcap_requirement(self):
         if not conf.use_npcap:
             raise OSError("This operation requires Npcap.")
         if self.raw80211 is None:
-            dot11adapters = next(iter(_vbs_exec_code("""WScript.Echo CreateObject("WScript.Shell").RegRead("HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\npcap\\Parameters\\Dot11Adapters")""")))
-            self.raw80211 = ("\\Device\\" + self.guid).lower() in dot11adapters.lower()
+            try:
```

Comment:
Could you add a comment to explain what this code do ?

Reply:
Done !